Controller/TutorialController.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace SumoCoders\FrameworkExampleBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Validator\Constraints\DateTime;
+
+class TutorialController extends Controller
+{
+    /**
+     * @Route("/tutorial/datepicker")
+     * @Template()
+     * @param Request $request
+     * @return array
+     */
+    public function datePickerAction(Request $request)
+    {
+        $defaultData = array(
```

Comment:
ik zou da in een formType class steken. Het is mss maar een voorbeeld, maar het zou leuk zijn om in de voorbeelden ook best practices te gebruikeN;